x-pack/plugin/sql/src/main/resources/org/elasticsearch/xpack/sql/plugin/sql_whitelist.txt

```diff
@@ -4,7 +4,14 @@
 # you may not use this file except in compliance with the Elastic License.
 #
 
-# This file contains a whitelist for SQL specific utilities available inside SQL scripting
+# This file contains a whitelist for SQL specific utilities and classes available inside SQL scripting
+
+#### Classes
+
+class org.elasticsearch.xpack.sql.expression.function.scalar.geo.GeoShape {
```

Comment:
ShapeBuilder is using GeoJSON for toXContent() and pretty useless debugging output in toString(), which complicates the serialization story sinice we need to use WKT in both. At the moment this is the whole purpose of this wrapper. Basically I am trying to avoid adding a whole bunch of 
```
if (obj instanceof ShapeBuilder) { 
   ... use different serialization for ShapeBuilder ... 
} else {
   ... normal serialization for pretty much everything else ...
}
```

I might also need to fold geo_point into this (not sure about this yet).